Sources/UIKit/ViewModelCollectionViewCell.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  ViewModelCollectionViewCell.swift
+//  Alicerce
+//
+//  Created by Luís Portela on 26/05/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import UIKit
+
+open class ViewModelCollectionViewCell<CellViewModel>: CollectionViewCell, ReusableViewModelView {
+    public typealias ViewModel = CellViewModel
+
+    open var viewModel: CellViewModel? {
+        didSet {
+            setUpBindings()
+        }
+    }
+
+    open func setUpBindings() {
+        fatalError("💥 forgot to override? 💣")
```

Comment:
For consistency sake, let's try and use the same error messages for un-overridden methods:

So far we've been using this one:

`fatalError("💥 Did you forget to override the method? 😱")`

👮